safaribooks.py

```diff
@@ -894,7 +894,7 @@ def create_content_opf(self):
             escape(self.book_info["description"]),
             subjects,
             ", ".join(escape(pub["name"]) for pub in self.book_info["publishers"]),
-            escape(self.book_info["rights"]),
+            escape(self.book_info["rights"]) if self.book_info["rights"] else "",
```

Comment:
Resolves 
```
[-] Creating EPUB file...
[#] Unhandled Exception: 'NoneType' object has no attribute 'replace' (type: AttributeError)
[+] Please delete all the `<BOOK NAME>/OEBPS/*.xhtml` files and restart the program.
[!] Aborting...
``` 
raised due to calling `escape(self.book_info["rights"])` when `self.book_info["rights"]` is None. Occurs with book ID 9780814429518 and 0738437395, for example.